pkg/crc/constants/constants_windows.go

```diff
@@ -1,17 +1,25 @@
 package constants
 
-import "path/filepath"
+import (
+	"os"
+	"path/filepath"
+)
 
 const (
-	OcExecutableName          = "oc.exe"
-	PodmanExecutableName      = "podman.exe"
-	AdminHelperExecutableName = "admin-helper-windows.exe"
-	TrayExecutableName        = "tray-windows.exe"
-	DaemonServiceName         = "CodeReady Containers"
-	TrayShortcutName          = "tray-windows.lnk"
+	OcExecutableName            = "oc.exe"
+	PodmanExecutableName        = "podman.exe"
+	AdminHelperExecutableName   = "admin-helper-windows.exe"
+	TrayExecutableName          = "tray-windows.exe"
+	TrayShortcutName            = "tray-windows.lnk"
+	DaemonBatchFileName         = "crc-daemon-autostart.bat"
+	DaemonPSscriptName          = "launch-crc-daemon.ps1"
+	DaemonBatchFileShortcutName = "crc-daemmon-autostart.bat.lnk"
 )
 
 var (
-	TrayExecutableDir  = filepath.Join(CrcBinDir, "tray-windows")
-	TrayExecutablePath = filepath.Join(TrayExecutableDir, TrayExecutableName)
+	StartUpFolder       = filepath.Join(os.Getenv("USERPROFILE"), "AppData", "Roaming", "Microsoft", "Windows", "Start Menu", "Programs", "Startup")
+	TrayExecutableDir   = filepath.Join(CrcBinDir, "tray-windows")
+	TrayExecutablePath  = filepath.Join(TrayExecutableDir, TrayExecutableName)
+	DaemonBatchFilePath = filepath.Join(CrcBinDir, DaemonBatchFileName)
+	DaemonPSscriptPath  = filepath.Join(CrcBinDir, DaemonPSscriptName)
```

Comment:
PSScript, as before,  although this double PS can be avoided if internally named Posh.